### PR TITLE
doc: support strategy

### PR DESCRIPTION
**Pull request by andycheng123**

### CODE REVIEW

File: doc/topics/back-testing.md [hunk 1]
```diff
@@ -16,6 +19,8 @@ backtest:
   - BTCUSDT
   
   account:
+    makerCommission: 15  # Commission rate of maker trade, in ‱, i.e., 1 means 0.01% 
+    takerCommission: 15  # Commission rate of taker trade, in ‱ i.e., 1 means 0.01%
```
Comment (c9s) [inline doc/topics/back-testing.md:23]: this is out-dated and should be removed.

Comment (andycheng123) [reply doc/topics/back-testing.md:23]: But i still see commission fields in backtest result?

Comment (c9s) [reply doc/topics/back-testing.md:23]: we will remove it, because it is not consistent.

Comment (andycheng123) [reply doc/topics/back-testing.md]: Got it. Removed

**Review by c9s — APPROVED** [review]
Thanks!

**Pull request merged**